aiida/cmdline/params/options/__init__.py

```diff
@@ -280,6 +280,11 @@ def active_process_states():
     type=click.Choice(['id', 'ctime']), default='ctime', show_default=True,
     help='Order the entries by this attribute.')
 
+ORDER_DIR = OverridableOption(
+    '-D', '--order-dir', 'order_dir',
```

Comment:
Let's not use an abbreviation here, i.e. let's use `--order-direction`, especially since the flags are tab completed